### PR TITLE
Add dracula theme

### PR DESCRIPTION
**Pull request by loewenheim**
I based this as closely as I could on the [specification](https://spec.draculatheme.com/).

### CODE REVIEW

File: runtime/themes/dracula.toml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+magenta = "#ff79c6"
+white = "#f8f8f2"
+bright_black = "#6272a4"
+bright_red = "#ff6e6e"
+bright_green = "#69ff94"
+bright_yellow = "#ffffa5"
+bright_blue = "#d6acff"
+bright_magenta = "#ff92df"
+bright_cyan = "#a4ffff"
+bright_white = "#ffffff"
```
Comment (archseer) [inline runtime/themes/dracula.toml:57]: If this is unused, you can go ahead and drop it

File: runtime/themes/dracula.toml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+"ui.cursor.primary" = { fg = "black", bg = "cyan", modifier = ["dim"] }
+"ui.cursor.match" = { fg = "green", modifiers = ["underlined"] }
+"ui.linenr" = { fg = "comment" }
+"ui.linenr.selected" = { fg = "foreground" }
+"ui.statusline" = { fg = "foreground", bg = "background_dark" }
+"ui.selection" = { fg = "black", bg = "purple", modifiers = ["dim"] }
+"ui.selection.primary" = { fg = "black", bg = "pink" }
+"ui.text" = { fg = "foreground" }
+"error" = { fg = "red" }
+"warning" = { fg = "cyan" }
```
[the remark this answers is not in the document]
Comment (loewenheim) [reply runtime/themes/dracula.toml:29]: Thank you for the reminder that I still needed to deal with these. Unfortunately I can't figure out what "menu" does.

[the remark this answers is not in the document]
Comment (CptPotato) [reply runtime/themes/dracula.toml:29]: > Thank you for the reminder that I still needed to deal with these. Unfortunately I can't figure out what `ui.menu` does.
> 
> ETA: The same goes for `ui.cursor.insert` and `ui.info`.

`ui.cursor.insert` is the color of the cursor in insert mode. It only works when `ui.cursor.primary` isn't set, though.
It's the same case for `ui.cursor.select` (-> visual mode).

`info` (not `ui.info` afaik) is used for infos such as suggestions and other lints from the LSP.

I wasn't able to find out what `ui.menu` refers to, though :/

[the remark this answers is not in the document]
Comment (archseer) [reply runtime/themes/dracula.toml:29]: `ui.menu` is used for the completion menu, and the picker "menu" (the left pane with the file list)

**Pull request merged**